autogpt/autogpt/app/agent_protocol_server.py

```diff
@@ -99,7 +99,7 @@ async def start(self, port: int = 8000, router: APIRouter = base_router):
         script_dir = os.path.dirname(os.path.realpath(__file__))
         frontend_path = (
             pathlib.Path(script_dir)
-            .joinpath("../../../../frontend/build/web")
+            .joinpath("../../frontend/build/web")
```

Comment:
this raises a linting error, probably because it now fits on one line?

Reply:
also I think you removed one `../` too many
```suggestion
            .joinpath("../../../frontend/build/web")
```